AssetManagementTestProject/Asserter/Asserter.cs

```diff
@@ -15,11 +16,34 @@ public class Asserter : WebDriverAction
     public Asserter() : base()
     {
     }
+    #region URL FLUENT ASSERTION
+    public void AssertUrlEquals(string actual, string expected)
+    {
+        AssertEquals(actual, expected);
+    }
+    #endregion
+
+    #region ELEMENTS DISPLAY FLUENT ASSERTION
+    public void AssertElementIsDisplayed(string locator)
+    {
+        IsElementDisplayed(locator).Should().BeTrue();
+    }
+
+    public void AssertElementsAreDisplayed(List<string> locators)
+    {
+        foreach(string locator in locators)
+        {
+            AssertElementIsDisplayed(locator);
+        }
+    }
```

Comment:
Assert each elem is enough for clearer code, no need to iterate through each elem in a list